drydock/templates/drydock/k8s/ingress/extra-hosts.yml

```diff
@@ -0,0 +1,37 @@
+{%- for host in DRYDOCK_INGRESS_EXTRA_HOSTS %}
+{%- if host != MFE_HOST %}
```

Comment:
I don't like this `if` statement too much. can we just trust the MFE host is not going to be added to the extra hosts?